atlassian/bitbucket/cloud/repositories/pullRequests.py

```diff
@@ -0,0 +1,312 @@
+            yield Participant(participant, **self._new_session_args)
+
+        return
+
+    def reviewers(self):
+        """ Returns a generator object of reviewers """
+        for reviewer in self.get_data("reviewers"):
+            yield User(None, reviewer, **self._new_session_args)
+
+        return
+
+    def comment(self, raw_message):
+        """
+        Commenting the pull request in raw format
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/comments#post
+        """
+        if not raw_message:
+            raise ValueError("No message set")
+
+        data = {
+            "content": {
+                "raw": raw_message,
+            }
+        }
+
+        return self.post("comments", data)
+
+    def approve(self):
+        """
+        Approve a pull request if open
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/approve#post
+        """
+        self._check_if_open()
+        data = {"approved": True}
+        return self.post("approve", data)
+
+    def unapprove(self):
+        """
+        Unapprove a pull request if open
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/approve#delete
+        """
+        self._check_if_open()
+        return self.delete("approve")
+
+    def request_changes(self):
+        """
+        Request changes for the pull request if open
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/request-changes#post
+        """
+        self._check_if_open()
+        data = {"request-changes": True}
+        return self.post("request-changes", data)
+
+    def unrequest_changes(self):
+        """
+        Request changes for the pull request if open
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/request-changes#delete
+        """
+        self._check_if_open()
+        return self.delete("request-changes")
+
+    def decline(self):
+        """
+        Decline a pull request
+
+        API docs: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/decline
+        """
+        self._check_if_open()
+        return self.post("decline")
+
+    def merge(self, merge_strategy=None, close_source_branch=None):
+        """
+        Merges the pull request if it's open
+        :param merge_strategy: string:  Merge strategy (one of "merge_commit", "squash", "fast_forward")
```

Comment:
Better reference the constants in the documentation, e.g. `self.MERGE_COMMIT`

Reply:
done.